### ISSUE TITLE
Multivec track doesn't move in sync with other tracks

### ISSUE THREAD

**Issue by pkerpedjiev** [hms-dbmi/higlass#282]
_Commit_: 

#### Steps to reproduce

1. Load the example from the viewconf here: 

http://higlass.io/app/?config=L4nKi6eGSzWOpi-rU2DAMA

#### Observed behavior

![jul-08-2018 23-01-54](https://user-images.githubusercontent.com/2143629/42428715-2004a9b2-8303-11e8-9549-c58fec9600e2.gif)
 

#### Expected behavior

Multivec moves in sync with other track.

**Comment by pkerpedjiev** [comment]
The issue is the following change which was introduced to accommodate the latest version of the multivec track:

```
 export const OneDimensionalMixin = Mixin(superclass => class extends superclass {

   initTile(tile) {
-    console.log('init');
     this.localColorToHexScale();

     this.unFlatten(tile);
     this.maxAndMin.max = tile.maxValue;
     this.maxAndMin.min = tile.minValue;
-    this.renderTile(tile);
+
     if (tile.matrix) {
       // update global max and min if necessary
       (this.maxAndMin.max === null || tile.maxValue > this.maxAndMin.max) ?
@@ -19,27 +18,38 @@ export const OneDimensionalMixin = Mixin(superclass => class extends superclass
       (this.maxAndMin.min === null || tile.minValue < this.maxAndMin.min) ?
         this.maxAndMin.min = tile.minValue : this.maxAndMin.min;
     }
+
+    this.renderTile(tile);
+    //this.rescaleTile(tile);
+
+    const visibleAndFetched = this.visibleAndFetchedTiles();
+    visibleAndFetched.map(a => {
+      this.rescaleTile(tile);
+    });
+
   }
``` 

It needs to be removed until the new version of the multivec track is released.

**Comment by pkerpedjiev** [comment]
Should be fixed in v0.2.0-alpha.2.
